hw/chip/mimxrt10xx/family.c

```diff
@@ -283,25 +273,17 @@ void board_init(void)
     CLOCK_EnableClock(kCLOCK_Iomuxc);
 
     // LED
-    IOMUXC_SetPinMux(LED_PINMUX, 0U);
-    IOMUXC_SetPinConfig(LED_PINMUX, 0x10B0U);
-
+    IOMUXC_SetPinMux(PIN_LED, PIN_LED_MUX);
+    IOMUXC_SetPinConfig(PIN_LED, PIN_LED_CFG);
     gpio_pin_config_t led_config = { kGPIO_DigitalOutput, 0, kGPIO_NoIntmode };
-    GPIO_PinInit(LED_PORT, LED_PIN, &led_config);
+    GPIO_PinInit(LED_GPIO_PORT, LED_GPIO_PIN, &led_config);
     board_led_write(false);
 
-    // Button
-    //IOMUXC_SetPinMux( BUTTON_PINMUX, 0U);
-    //IOMUXC_SetPinConfig(BUTTON_PINMUX, 0x01B0A0U);
-    //gpio_pin_config_t button_config = { kGPIO_DigitalInput, 0, kGPIO_IntRisingEdge, };
-    //GPIO_PinInit(BUTTON_PORT, BUTTON_PIN, &button_config);
-
     // UART
-    IOMUXC_SetPinMux( UART_TX_PINMUX, 0U);
-    IOMUXC_SetPinMux( UART_RX_PINMUX, 0U);
-    IOMUXC_SetPinConfig( UART_TX_PINMUX, 0x10B0u);
-    IOMUXC_SetPinConfig( UART_RX_PINMUX, 0x10B0u);
-
+    IOMUXC_SetPinMux( PIN_UART_TX, PIN_UART_TX_MUX);
+    IOMUXC_SetPinMux( PIN_UART_RX, PIN_UART_RX_MUX);
```

Comment:
The extra spaces here are by mistake, you can remove them

Reply:
Fixed.